stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/proto/ServiceProtoConstants.java

```diff
@@ -52,10 +52,6 @@
     public static final String ANN_SERVICE_CONFIG_FQN = PROTOCOL_STRUCT_PACKAGE_GRPC + ":ServiceConfig";
     public static final String ANN_RESOURCE_CONFIG = "ResourceConfig";
 
-    public static final String SERVICE_CONFIG_RPC_ENDPOINT = "name";
-    public static final String SERVICE_CONFIG_CLIENT_STREAMING = "clientStreaming";
-    public static final String SERVICE_CONFIG_SERVER_STREAMING = "serverStreaming";
-
     public static final String RESOURCE_CONFIG_STREAMING = "streaming";
     public static final String RESOURCE_CONFIG_REQUEST_TYPE = "requestType";
     public static final String RESOURCE_CONFIG_RESPONSE_TYPE = "responseType";
```

Comment:
```suggestion
```

Reply:
We can remove resource config annotation completely

Reply:
I completely removed the resource config